src/main/java/seedu/address/logic/commands/StatisticsCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.HashMap;
+
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+import seedu.address.model.Model;
+import seedu.address.model.person.JobCode;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Tag;
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class StatisticsCommand extends Command {
+
+    public static final String COMMAND_WORD = "statistics";
+
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        // create a filtered list of all the predicate persons
+        // then create iteration
+        ObservableList<Person> lastShownList = model.getFilteredPersonList();
+        FilteredList<Person> fullContactList = new FilteredList<>(lastShownList);
+        int numOfApplicants = 0;
+        HashMap<JobCode, Integer> jobDict = new HashMap<JobCode, Integer>();
+        HashMap<Tag, Integer> tagDict = new HashMap<Tag, Integer>();
+        for (Person person : fullContactList) {
+            numOfApplicants++;
+            JobCode jobCode = person.getJobCode();
+            jobDict.put(jobCode, jobDict.getOrDefault(jobCode, 0) + 1);
```

Comment:
Can consider creating a helper class that stores the statistics of the tags for the job code. E.g each JobCodeStats have the separate tag properties.